python/datashaper/datashaper/verbs/copy.py

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright (c) Microsoft. All rights reserved.
+# Licensed under the MIT license. See LICENSE file in the project.
+#
+"""Copy verb implementation."""
+
+from typing import Any
+
+import pandas as pd
+
+from .decorators import OutputReturnType, apply_decorators, inputs, outputs, verb
+
+
+def copy(
+    table: pd.DataFrame,
+    to: str,
+    column: str,
+    **_kwargs: Any,
+) -> pd.DataFrame:
+    """Copy verb implementation."""
+    table[to] = table[column]
+    return table
+
+
+apply_decorators(
```

Comment:
This feels like a layer of abstraction too far - just raw annotations on the verbs would be much more clear than sending to a wrapper function that probably doesn't actually reduce any lines of code

Reply:
Per our conversation, applying decorators as a separate expression like this allows us to export the base verb functions as standalone utilities. The @input and @output decorators fundamentally change these verbs for use with datashaper's verb semantics.